estate_account/models/estate_property.py

```diff
@@ -0,0 +1,27 @@
+from odoo import Command, models
+
+
+class Property(models.Model):
+    _inherit = "estate.property"
+
+    def sell(self):
+        for record in self:
+            values = {
+                'partner_id': record.buyer_id.id,
+                'move_type': 'out_invoice',
+                'invoice_line_ids': [
+                    Command.create({
+                        'name': '6% of the selling price',
+                        'quantity': 1,
+                        'price_unit': 0.06 * record.selling_price,
+                    }),
+                    Command.create({
+                        'name': 'Administrative fees',
+                        'quantity': 1,
+                        'price_unit': 100,
+                    }),
```

Comment:
I think you are missing the important line, the one with the main amount. So far, you have the tax, and the admin fees